docs/user-guide/advanced-configuration/mesh-ingress.md

```diff
@@ -0,0 +1,143 @@
+# Mesh Ingress
+
+The mesh ingress allows user to peer external execution and hop nodes into the AWX control plane.
+This guide focuses on how to enable and configure the mesh ingress.
+For more information about what is remote execution node and hop node and how to create them see AWX Documentation on external remote execution node and hop node. <TODO: add AWX doc link>
+
+## Pre-requists
+
+- AWX operator version> 2.11.0
+- AWX > 23.8.0
+
+## Deploy and configure AWXMeshIngress
+
+### On Red Hat OpenShift with operator managed Route
+
+To deploy an mesh ingress on OpenShift create the AWXMeshIngress resource.
+
+Example:
+
+```yaml
+---
+apiVersion: awx.ansible.com/v1alpha1
+kind: AWXMeshIngress
+metadata:
+  name: <mesh ingress name>
+spec:
+  deployment_name: <awx instance name>
+```
+
+### User managed Ingress
+
+UNDER CONSTRUCTION (contribution welcome)
+
+### Operator managed Ingress
+
+UNDER CONSTRUCTION (contribution welcome)
+
+### Deploy and configure AWXMeshIngress via IngressRouteTCP
+
+UNDER CONSTRUCTION (contribution welcome)
+
+## Validating setup of Mesh Ingress
+
+After AWXMeshIngress has been successfully created a new Instance with the same name will show up in AWX Instance UI
+
+![mesh ingress instance on AWX UI](mesh-ingress-instance-on-awx-ui.png)
+
+The Instance should have at least 2 Listener Address
+
+![mesh ingress instance listener address on awx ui](mesh-ingress-instance-listener-address-on-awx-ui.png)
+
+When selecting peer for new instance the mesh ingress instance should now be present as a option.
+![peering to mesh ingress on awx ui](peering-to-mesh-ingress-on-awx-ui.png)
+
+For more information about how to create external remote execution and hop node and configuring the mesh. See AWX Documentation on managing external remote execution node and hop node. <TODO: add AWX doc link>
```

Comment:
Link to: https://ansible.readthedocs.io/projects/awx/en/latest/administration/instances.html#add-an-instance